src/Common/Commands.Common/Attributes/Ps1XmlAttribute.cs

```diff
@@ -23,11 +23,24 @@ public enum ViewControl
         All = Table | List,
     }
 
-    [AttributeUsage(AttributeTargets.Property | AttributeTargets.Field)]
+    [AttributeUsage(AttributeTargets.Property | AttributeTargets.Field, AllowMultiple = true)]
     public sealed class Ps1XmlAttribute : Attribute
     {
         public string Label { get; set; }
 
         public ViewControl Target { get; set; } = ViewControl.Table;
+
+        public string ScriptBlock { get; set; }
+
+        public bool GroupByThis { get; set; }
+
+        public uint TableColumnWidth { get; set; }
+
+        public uint Position { get; set; } = Ps1XmlConstants.DefaultPosition;
```

Comment:
so, what happens if :
* A user specified the same position on multiple properties in the same class?

* Not all properties in the class have a position specified?